10print/10PrintinJS.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
rename the directory (this in unclear), and create a readme explaining what you're trying to do with this program.